classes/site/Version.inc.php

```diff
@@ -284,6 +285,47 @@ function getVersionString($numeric = true) {
 
 		return $numericVersion;
 	}
+
+	/**
+	 * Checks if this version is compatible with given version
+	 * Returns:
+	 * 		true iff the version given is compatible with this version
+	 * 		false iff the version given is incompatible with this version
+	 * @param $version string/Version the version to compare against
+	 * @return boolean
+	 */
+	function isCompatible($version) {
+		$versionParser = new VersionParser();
+		$semver = new Sem();
+		$parsed = $versionParser->parseConstraints($version);
+		$systemVersion = $this->getVersionString();
+		if ($parsed instanceof Composer\Semver\Constraint\MultiConstraint) {
+			$success = false;
+			$localFailure = false;
+			$multiConstraints = $parsed->getConstraints();
+			foreach ($multiConstraints as $constraints) {
+				$localFailure = false;
+				if ($constraints instanceof Composer\Semver\Constraint\MultiConstraint) {
+					$constraintsLV2 = $constraints->getConstraints();
```

Comment:
Hopefully superseded, but `$constraintsLV2` is not camelCase compliant.   To describe `$constraintsLevelTwo`, spell it out for future readers.

Reply:
Superseded by the above.